var/spack/repos/builtin/packages/charm/package.py

```diff
@@ -51,18 +51,8 @@ class Charm(Package):
     # Ignore compiler warnings while configuring
     patch("strictpass.patch")
 
-    # Communication mechanisms (choose exactly one)
-    # TODO: Support Blue Gene/Q PAMI, Cray GNI, Cray shmem, CUDA
```

Comment:
I would keep these comments so we know what `backend` means and we know that there is still work to be done.